packages/lib/OgImages.tsx

```diff
@@ -83,16 +93,11 @@ export const constructGenericImage = ({ title, description }: GenericImageProps,
   return encodeUri ? encodeURIComponent(url) : url;
 };
 
-const Wrapper = ({
-  children,
-  variant = "light",
-}: {
-  children: React.ReactNode;
-  variant?: "light" | "dark";
-}) => (
+const Wrapper = ({ children, variant = "light", rotateBackground }: WrapperProps) => (
   <div tw="flex w-full h-full">
     <img
       tw="flex absolute left-0 top-0 w-full h-[110%]"
+      style={rotateBackground ? { transform: "rotate(180deg)" } : undefined}
```

Comment:
The Satori TW property doesn't support rotate, that's why there's an inline style declaration here.